test/ctia/entity/incident_test.clj

```diff
@@ -313,7 +315,7 @@
                        (testing (pr-str test-id)
                          (let [{:keys [parsed-body] :as raw}
                                (search-th/search-raw app :incident {:sort_by
-                                                                    (format "severity:%1$s,created:%1$s"
+                                                                    (format "severity:%1$s,timestamp:%1$s"
```

Comment:
@frenchy64 I think that it fixes https://github.com/advthreat/iroh/issues/6619
wdyt?

Reply:
Oh! Yes, that looks like the problem.

Reply:
And I do not know why, but I hit it consistently locally with this branch.